src/components/form/range/range_slider.tsx

```diff
@@ -23,6 +23,10 @@ export type EuiRangeSliderProps = InputHTMLAttributes<HTMLInputElement> &
     max: number;
     step?: number;
     compressed?: boolean;
+    /*
+     * Will only show if `showInput = inputWithPopover`
+     */
+    isLoading?: boolean;
```

Comment:
This comment isn't showing in the props table: 

<img width="844" alt="Screenshot 2022-02-17 at 20 05 30" src="https://user-images.githubusercontent.com/2750668/154562140-81c39f94-6808-42bb-8a2a-a4db8962de51.png">

Reply:
Needs the JSDoc comment format:

```suggestion
    /**
     * Will only show if `showInput = inputWithPopover`
     */
    isLoading?: boolean;
```